ws-nextjs-app/pages/[service]/new_live/[id]/LivePageLayout.tsx

```diff
@@ -88,32 +88,34 @@ const LivePage = ({
           title={title}
           description={description}
         />
-        <div css={styles.wrapper}>
-          <Summary summaryBlocks={summaryContent?.model.blocks} />
-          {liveTextStream.content && (
-            <PostsList postData={liveTextStream.content} />
-          )}
-          <pre css={styles.code}>
-            <Heading level={4}>Headers</Heading>
-            {bbcOrigin && (
+        <div css={styles.outerGrid}>
+          <div css={styles.leftSection}>
+            <Summary summaryBlocks={summaryContent?.model.blocks} />
+          </div>
+          <div css={styles.rightSection}>
+            <Stream streamContent={liveTextStream.content} />
+            <pre css={styles.code}>
+              <Heading level={4}>Headers</Heading>
+              {bbcOrigin && (
+                <p>
+                  bbc-origin: <span>{bbcOrigin}</span>
+                </p>
+              )}
               <p>
-                bbc-origin: <span>{bbcOrigin}</span>
+                bbc-adverts:{' '}
+                <span>{showAdsBasedOnLocation ? 'true' : 'false'}</span>
               </p>
-            )}
-            <p>
-              bbc-adverts:{' '}
-              <span>{showAdsBasedOnLocation ? 'true' : 'false'}</span>
-            </p>
-          </pre>
-          <pre css={styles.code}>{JSON.stringify(pageData, null, 2)}</pre>
-          <Pagination
-            activePage={activePage}
-            pageCount={pageCount}
-            pageXOfY="Page {x} of {y}"
-            previousPage="Previous Page"
-            nextPage="Next Page"
-            page="Page"
-          />
+            </pre>
+            <pre css={styles.code}>{JSON.stringify(pageData, null, 2)}</pre>
+            <Pagination
+              activePage={activePage}
+              pageCount={pageCount}
+              pageXOfY="Page {x} of {y}"
+              previousPage="Previous Page"
+              nextPage="Next Page"
+              page="Page"
+            />
```

Comment:
I'd maybe get rid of these now, unless they are helpful in testing?

Reply:
I think the headers and page data are useful for testing/debugging so I might leave those in. I'll defo get rid of the pagination for now